.github/CONTRIBUTING-en-US.md

```diff
@@ -1,42 +1,97 @@
+#### Issues
 
-We also have a [discussion forum](https://github.com/okfn-brasil/querido-diario-comunidade/discussions) where you can contribute ideas related to the project.
+On the **Issues** tab of the chosen repository, there are tasks or issues
+specific to that repository. Check to see if any of them interest you.
+- When a task is suitable for beginners, it will be labeled as `good first issue`.
+- When you find a task that looks interesting to you, comment that you will do that task. That way, other folks will know that you're already working on it, and can offer to help you or take on other tasks that haven't been worked on yet.
+- Whether or not you've already chosen an *issue* to work on, you can also comment on relevant questions regarding the task.
 
-And a [community on Discord](https://bit.ly/discord-ok), if you want to ask questions faster or talk to people who are also interested in Open Knowledge Brasil's civic innovation projects.
+#### Milestones
+
+If there are no open *issues* or none interest you, look at the **Milestones** of the repository you want to contribute to. These indicate the goals to be achieved. Exploring the project files looking for areas for improvement that would help the repository reach a *milestone* is an excellent way of contributing.
+- When you find something, open an *issue* describing the potential improvement. If you are also planning to work on that task, add that to the issue.
+
+#### Discussions
+
+In case you have an idea or suggestion for the project's direction, open a **Discussion**.
+- As *Querido Diário* is made up of many repositories and discussions tend to be transversal, it is **a management's decision** to keep them all centralized. So, to start or follow discussions, just look at the tab [`Discussion` in `querido-diario-community`](https://github.com/okfn-brasil/querido-diario-comunidade/discussions).
+
+#### Community chat
+
+Join our [community](https://go.ok.org.br/discord). There, you'll be able to ask
+quick questions, interact with the project maintainers, learn about how can you
+help and chat with other *Querido Diário* contributors.
+
+### Interacting with reviewers
+
+- Be patient, and give reviewers some time. In addition to contribution reviews being complex
+and demanding time and attention by themselves, a reviewer can be - just like you - another
+contributor, who's volunteering their personal time when possible, or has other priorities
+before being able to review your contribution.
+- Whether in the form of code or messages with questions and suggestions, try to be objective,
+but giving details when necessary, to facilitate interactions.
+- During the reviewing process, a reviewer can open *threads* requesting improvements to
+your contribution. In general, it is the reviewers' role to decide when a *thread* has ended.
```

Comment:
Esta linha aqui é relativa a marcar as discussões na interface do GH como resolvidas/resolved? Eu tinha achado que era algo mais abstrato, quando estava traduzindo, porque não estou acostumada com esse uso, mas fui pescar em outro PR como era o processo de incorporar feedback de vocês, e aí caiu a ficha que provavelmente era a isso que se referia...

Se for, será que faria sentido colocar uma imagem pra ajudar a visualizar? Ou é algo mais universal, que as pessoas aprendem com tutoriais de git?

Reply:
É exatamente isso mesmo :)

Prefiriria que a gente colocasse imagem de interface (que não temos controle) em último caso para não termos problemas com estilos mudando ou diferenças entre dispositivos.

Mas não, não é algo que se aprende nos tutoriais de git normalmente. Deixar o texto mais explícito funcionaria pra você? Algo como:

During the reviewing process, a reviewer can open *threads* requesting improvements to your contribution. In general, it is the reviewers' role to decide when a *thread* has ended **and mark it as "resolved" in the interface. That will mean that both of you agree with the implemented modifications and the contribution can move on.**

Reply:
> É exatamente isso mesmo :)
> 
> Prefiriria que a gente colocasse imagem de interface (que não temos controle) em último caso para não termos problemas com estilos mudando ou diferenças entre dispositivos.

Beleza! Fatos, por vezes eu me passo nessas paradas...
 
> Mas não, não é algo que se aprende nos tutoriais de git normalmente. Deixar o texto mais explícito funcionaria pra você? Algo como:
> 
> During the reviewing process, a reviewer can open _threads_ requesting improvements to your contribution. In general, it is the reviewers' role to decide when a _thread_ has ended **and mark it as "resolved" in the interface. That will mean that both of you agree with the implemented modifications and the contribution can move on.**

Acho que fica melhor! Vou incorporar, acho que só vou acrescentar um "git hub" no interface :P

Reply:
Quando fecharmos aqui, então, devo atualizar o Contributing em português, pra ficarem de acordo?